src/libraries/Common/tests/TestUtilities/System/PlatformDetection.Windows.cs

```diff
@@ -30,6 +30,7 @@ public static partial class PlatformDetection
         public static bool IsNotWindows8x => !IsWindows8x;
         public static bool IsNotWindowsNanoServer => !IsWindowsNanoServer;
         public static bool IsNotWindowsServerCore => !IsWindowsServerCore;
+        public static bool IsNotWindowsNanoAndServerCore => IsNotWindowsNanoServer && IsNotWindowsServerCore;
```

Comment:
Nit, I think this should be `IsNotWindowsNanoNorServerCore` or `IsNeitherWindowsNanoNorServerCore` or `IsNotWindowsNanoAndIsNotServerCore`. It sounds like there's an OS named WindowsNanoAndServerCore, and we're not on that..

Reply:
Sounds good, thanks for the feedback.